provision/provision-helpers.sh

```diff
@@ -204,3 +223,45 @@ function get_config_keys() {
   echo "${value:-$2}"
 }
 export -f get_config_keys
+
+#
+# hook engine
+#
+vvv_add_hook() {
+  if [[ "${1}" =~ [^a-zA-Z_] ]]; then
+    echo "Disallowed hookname"
```

Comment:
maybe use vvv_warn or something that help with debug?

Reply:
Agreed, perhaps also mention the hook name in quotes so it can be tracked

Reply:
Perhaps this?

```suggestion
    vvv_warn "Invalid hookname '${1}', hooks must only contain the characters A-Z and a-z"
```